sites/vehicleservicepros.com/server/routes/website-section.js

```diff
@@ -15,13 +16,22 @@ const channelAliases = [
   'vehicles',
   'distributors',
   'industry-news',
+  'diagnosticNetworkFeed',
 ];
 
 module.exports = (app) => {
   app.get('/:alias(leaders)', withWebsiteSection({
     template: leaders,
     queryFragment: leadersFragment,
   }));
+  app.get('/:alias(in-the-bay)', withWebsiteSection({
+    template: diagnosticNetworkFeed,
+    queryFragment,
+  }));
+  app.get('/:alias(covid-19)', withWebsiteSection({
```

Comment:
Is the covid-19 seection supposed to use this layout?

Reply:
yes these In The Bay and Covid-19